recipes/semi-ate-control-app/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "semi-ate-control-app" %}
+{% set version = "1.0.9" %}
+
+ 
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/semi-ate-control-app-{{ version }}.tar.gz
+  sha256: f7af3dfd14ae5940a9a2b2eeca050d1714cc5b841ff59d6f0b96c9dc9024e3d3
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - launch_control=ate_control_app.launch_control:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip >=3.8
+    - python
+  run:
+    - python >=3.8
+    - semi-ate-apps-common
+    - semi-ate-common
+    - transitions
+
+test:
+  imports:
+    - ate_control_app
+  commands:
+    - pip check
+    - launch_control --help
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/semi-ate-control-app/
+  summary: Shared helpers used by the different ate-apps, i.e. master-app, control-app, etc.)
```

Comment:
Looks like this is still what was copied from another recipe?

Reply:
Woops, you are right :upside_down_face: